packages/bpk-foundations-common/base/colors/sponsoredbanner.json

```
@@ -0,0 +1,15 @@
+{
+    "imports": ["./aliases.json"],
+    "global": {
+        "type": "color",
+        "category": "sponsoredbanner-colors"
```

Small nit, just for consistency:

```suggestion
        "category": "sponsored-banner-colors"
```